app/models/manageiq/providers/openshift/container_manager/orchestration_stack.rb

```diff
@@ -0,0 +1,56 @@
+class ManageIQ::Providers::Openshift::ContainerManager::OrchestrationStack < ManageIQ::Providers::ContainerManager::OrchestrationStack
+  require_nested :Status
+
+  belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ManageIQ::Providers::ContainerManager"
```

Comment:
I feel the class naming is strange. The template instantiating logic is in [base](https://github.com/ManageIQ/manageiq/blob/master/app/models/container_template.rb#L29) class in core repository, so the code is generic enough; then corresponding stack should be generic too, not limit to openshift.

To conform to our file structures for all other providers I feel we should move the actual logic of `ContainerTemplate` to a common provider folder such as `ManageIQ::Providers::ContainerManager`. Then we will have
`ManagerIQ::Providers::ContainerManager::ContainerTemplate`
`ManagerIQ::Providers::ContainerManager::OrchestrationStack`
`ManagerIQ::Providers::ContainerManager::OrchestrationStack::Status`